
Epoch: 1
Traceback (most recent call last):
  File "main.py", line 91, in <module>
    acc, auc, f1 = valid_func(model, loader_valid)
  File "/homes/n20darja/StageCesureCoeur/predict_tavi/Learning2D/train.py", line 61, in valid_func
    logits = feat.view(bs, 98).contiguous()
NameError: name 'bs' is not defined
Traceback (most recent call last):
  File "main.py", line 91, in <module>
    acc, auc, f1 = valid_func(model, loader_valid)
  File "/homes/n20darja/StageCesureCoeur/predict_tavi/Learning2D/train.py", line 61, in valid_func
    logits = feat.view(bs, 98).contiguous()
NameError: name 'bs' is not defined